lib/math.ts

```diff
@@ -55,3 +58,50 @@ export function FMAPowerOf10(xParam: number, p: number, zParam: number) {
   const resStr = xStr + ReflectApply(StringPrototypePadStart, zStr, [p, '0']);
   return sign * NumberParseInt(resStr, 10);
 }
+
+type UnsignedRoundingMode = 'half-even' | 'half-infinity' | 'half-zero' | 'infinity' | 'zero';
+
+export function GetUnsignedRoundingMode(
+  mode: Temporal.RoundingMode,
+  sign: 'positive' | 'negative'
+): UnsignedRoundingMode {
+  const index = +(sign === 'negative'); // 0 = positive, 1 = negative
```

Comment:
```suggestion
  const index = +(sign === 'negative') as 0 | 1; // 0 = positive, 1 = negative
```

TS doesn't know what you do! :-)